extensions/notebook/package.json

```diff
@@ -17,6 +17,11 @@
       "type": "object",
       "title": "%notebook.configuration.title%",
       "properties": {
+        "notebook.collapsedBookItems": {
+          "type": "boolean",
+          "default": true,
```

Comment:
The default should be false to maintain backward compatibility.

Reply:
This looks great 👍